web/source/keyboards/kmwkeyboards.ts

```diff
@@ -135,7 +135,7 @@ namespace com.keyman.keyboards {
      */
     registrationResolvers: {[timeoutID: number] : RegistrationPromiseTuple} = {};
 
-    languageListPromise: Promise<(KeyboardStub|ErrorStub)[]> = null;
+    languageListPromise: Promise<(KeyboardStub)[]|Error> = null;
```

Comment:
Please keep in mind that including `Error` here implies that the `then` clause should expect to handle the type.

`Error` is always expected for the `reject` -> `catch` path, so if `Error`s are only passed along through that mechanism, `|Error` should be removed from the generic specialization seen here.